reverse_strand.py

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/python3
+import argparse
+from argparse import RawTextHelpFormatter
+import bamnostic as bs
+
+description="Reverses the allele for variants that got mapped onto the reverse strand of the new genome, and prints everything to a new file\nAlso filters out the non-mapped"
+
+parser = argparse.ArgumentParser(description = description, formatter_class=RawTextHelpFormatter)
+parser.add_argument("-i", "--bam", help="bam file containing remapped variants ")
+parser.add_argument("-o", "--outfile", help="name of new file")
+args = parser.parse_args()
+
+bamfile=bs.AlignmentFile(args.bam, 'rb')
+outfile=open(args.outfile, 'w')
+
+#reverses the allele for variants that got mapped onto the reverse strand of the new genome, and prints everything into correct columns
```

Comment:
This loop seems to complement the bases, but does it also reverse the sequence? Or is it not needed?

Reply:
By sequence do you mean the reference genome? Or the read that got mapped?

Reply:
I guess @cyenyxe meant the read. Because we're only interested in SNPs for now, we only need to flip the strand for one specific nucleotide, so actually reverse-complementing the read is unnecessary for now